### PR TITLE
Make small edits to README

### PR DESCRIPTION
**Pull request by batpigandme**
* Reflect order in which packages are loaded.
* Minor copy edits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ devtools::install_github("tidyverse/dtplyr")
 
 ## Usage
 
-To use dtplyr, I recommend loading dplyr, dtplyr, and data.table:
+To use dtplyr, I recommend loading data.table, dtplyr, and dplyr:
```
Comment (hadley) [inline README.md]: I think you accidentally did these in the `.md`?

Comment (batpigandme) [reply README.md]: They're in the Rmd, too (I just hadn't rendered the md until the second commit).

Comment (batpigandme) [reply README.md]: https://github.com/tidyverse/dtplyr/pull/78/files#diff-5140cd39fa2897e01ffbfb8db0999940R49

**Comment by hadley** [comment]
Ok, must've missed that — thanks!

**Pull request merged**